src/app/app.component.ts

```diff
@@ -666,9 +671,157 @@ export class AppComponent implements OnInit {
+        this.comicsInReadingOrder.push(_.cloneDeep(this.comics[comicIndex]));
       });
     });
 
+    //  reposition the reading order comics horizontally
+    let horizontalReadingOrderPosition = 0;
+    this.globalVerticalPositionCounter = 0;
+    const latestReadingOrderVerticalHorizontalOffsets = {};
+    _.each(this.comicsInReadingOrder, (comic) => {
+      horizontalReadingOrderPosition += VISUAL_BLOCK_SIZE;
+      comic.containerStyles['left.px'] = horizontalReadingOrderPosition;
+
+      /*
+       * Look up the volume and series for this comic and
+       * throw errors for obvious problems before proceeding.
+       */
+      // tslint:disable-next-line: max-line-length
+      const currentReadingOrderSeriesVolume = this.readingOrderSeriesVolumes[_.findKey(this.readingOrderSeriesVolumes, { id: comic.seriesVolumeId })];
+      if (!currentReadingOrderSeriesVolume) {
+        throw new Error(comic.seriesVolumeId + ' not found');
+      }
+      // this is a copy of the other positioning
+      /*
+       * Vertical positioning ensures that each seriesVolume gets
+       * its own row on the page. The exception is if a seriesVolume
+       * has not had any new issues for a whole page width, then we
+       * allow the next seriesVolume that is looking for a row, to slot in.
+       *
+       * This is a two-step process.
+       * First, in this block, we either use the current position for the
+       * seriesVolume or we use the globalVerticalPositionCounter to go onto
+       * the last row.
+       * Step two documented below.
+       */
+      const isVerticalPositionAlreadyDefined = typeof currentReadingOrderSeriesVolume.verticalPosition !== 'undefined';
+      if (isVerticalPositionAlreadyDefined) {
+        comic.containerStyles['top.px'] = currentReadingOrderSeriesVolume.verticalPosition * VISUAL_BLOCK_SIZE;
+      } else {
+        currentReadingOrderSeriesVolume.verticalPosition = this.globalVerticalPositionCounter;
+        comic.containerStyles['top.px'] = this.globalVerticalPositionCounter * VISUAL_BLOCK_SIZE;
+        this.globalVerticalPositionCounter++;
+        newLabelNeeded = true;
+      }
+
+       /*
+       * Step two of vertical positioning:
+       * At this point, the seriesVolume has a row to use, but in this
+       * block we check if there is a row further up the page to slot into
+       * so we take up less vertical space.
+       */
+
+      /*
+       * The maximum horizontal offset allowed until we recycle the
+       * vertical position.
+       */
+      const horizontalClearanceLimit = comic.containerStyles['left.px'] - windowWidth;
+      if (
+        !latestReadingOrderVerticalHorizontalOffsets[currentReadingOrderSeriesVolume.verticalPosition] ||
+        (
+          // tslint:disable-next-line: max-line-length
+          latestReadingOrderVerticalHorizontalOffsets[currentReadingOrderSeriesVolume.verticalPosition].seriesVolumeId !== currentReadingOrderSeriesVolume.id &&
```

Comment:
Same comment about breaking up the line to remove the tslint disable

Reply:
I tried to read the documentation but I am unsure about what style to use here and above.